### PR TITLE
Add xeus-robot to the debugger documentation

### LINKED ISSUE #9634: Add xeus-robot to the Debugger documentation
Since the recently announced `xeus-robot` kernel also supports the JupyterLab visual debugger, we should consider adding it to the documentation:

image

https://jupyterlab.readthedocs.io/en/stable/user/debugger.html?#debugger

Blog post about `xeus-robot`: https://blog.jupyter.org/robotic-process-automation-with-jupyterlab-713b5630e457

### PR DESCRIPTION
**Pull request by jtpio**
## References

Fixes #9634 

## Code changes

Documentation changes.

## User-facing changes

In the documentation:

image

## Backwards-incompatible changes

None

### CODE REVIEW

File: docs/source/user/debugger.rst [hunk 1]
```diff
@@ -12,11 +12,17 @@ Requirements
...
+Here is a list of kernels that are known to be supporting the debug protocol:
+
+- `xeus-python <https://github.com/jupyter-xeus/xeus-python>`_: Jupyter kernel for the Python programming language
+- `xeus-robot <https://github.com/jupyter-xeus/xeus-robot>`_: Jupyter kernel for Robot Framework
+
+Other Jupyter Kernels can also support debugging and be compatible with the JupyterLab debugger
 by implementing the `Jupyter Debugger Protocol <https://jupyter-client.readthedocs.io/en/latest/messaging.html#debug-request>`_.
 
-To install ``xeus-python`` in a new ``conda`` environment:
+If you know of other kernels with support for debugging, please open a PR to add them to this list!
```
Comment (jasongrout) [inline docs/source/user/debugger.rst:23]:
```suggestion
If you know of other kernels with support for debugging, please open a PR to add them to this list.
```

I know you are excited about getting new kernels. To me this seems a bit much for documentation :).

File: docs/source/user/debugger.rst [hunk 2]
```diff
@@ -12,11 +12,17 @@ Requirements
...
+
+Here is an example on how to install ``xeus-python`` in a new ``conda`` environment:
```
Comment (jasongrout) [inline docs/source/user/debugger.rst:25]:
```suggestion
Here is an example of how to install ``xeus-python`` in a new ``conda`` environment:
```

**Review by jasongrout — APPROVED** [review]
I had a few comments, but other than that, looks good! Thanks!

File: docs/source/user/debugger.rst (hunk 1 above)
Comment (jtpio) [reply docs/source/user/debugger.rst:23]: Haha yes I think it was written with some kind of "blog post tone" to it, but it makes sense to follow the style of the documentation :)

Thanks!

**Comment by jtpio** [comment]
Thanks Jason for the review!

**Pull request merged**